### ISSUE TITLE
2 way Org Chart

### ISSUE THREAD

**Issue by airx910** [bumbeishvili/d3-organization-chart#60]
Is it possible to build a 2 way Org Chart. which to add a level of nodes on the top of the root(company) to show the owners/stockholder of it.

**Comment by bumbeishvili** [comment]
Yes, it's possible but will require significant additional work on top of the current code

**Comment by bumbeishvili** [comment]
I think yes, but will require significant amount of additional work

**Comment by fabianhofer8** [comment]
> Is it possible to build a 2 way Org Chart. which to add a level of nodes on the top of the root(company) to show the owners/stockholder of it.

Have you been successful?
